packages/hash/api/src/graphql/typeDefs/knowledge/comment.typedef.ts

```diff
@@ -84,6 +93,20 @@ export const persistedCommentTypedef = gql`
       tokens: [TextToken!]!
     ): PersistedComment!
 
+    """
+    Update an existing comment
+    """
+    updatePersistedComment(
+      """
+      Id of the comment being deleted
+      """
```

Comment:
If you are going to have a deletion-specific mutation, I suggest not asking for the timestamp. The server can set the timestamp. Otherwise people can fake the deletion time. I would also call it something else.

If this is going to be used for resolving comments as well, the comments here need updating to stop refering to 'the comment being deleted'

Reply:
This was fixed in e54545814ea2ab44fa446fadc368585a1f40cd9f by replacing the mutation.